dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPI.java

```diff
@@ -149,4 +150,19 @@ Optional<Rule> getRule(final Experiment experiment)
      * @return
      */
     boolean isAnyExperimentRunning() throws DotDataException;
+
+    /**
+     * Return the Experiment partial or total result.
+     *
+     * @param experiment
+     * @return
+     */
+    ExperimentResult getResult(final Experiment experiment);
```

Comment:
Rename to `getResults` and the object to `ExperimentResults`

Reply:
done https://github.com/dotCMS/core/pull/24034/commits/cb564daa0c443e886c8931149f0ef0a62935fc0f#diff-8bf1865184fd59c5c64882df72bd0c71a403a544777830acf250bbf54ea84d90R160